cdci_data_analysis/analysis/email_helper.py

```diff
@@ -395,19 +424,42 @@ def is_email_to_send_run_query(logger, status, time_original_request, scratch_di
                     times.append(float(f_name.split('_')[2]))
 
             time_last_email_submitted_sent = max(times)
-            time_from_last_submitted_email = time_.time() - float(time_last_email_submitted_sent)
+            time_from_last_submitted_email = time_check - float(time_last_email_submitted_sent)
             interval_ok = time_from_last_submitted_email > email_sending_job_submitted_interval
 
-        logger.info("email_sending_job_submitted: %s", email_sending_job_submitted)
         logger.info("interval_ok: %s", interval_ok)
+        log_additional_info_obj['interval_ok'] = interval_ok
+
+        status_ok = True
+        if status != 'submitted':
+            status_ok = False
+            logger.info(f'status {status} not a valid one for sending an email after a run_query')
+            if sentry_client is not None:
+                sentry_client.capture('raven.events.Message',
```

Comment:
oh, could you make a dispatcher config option to disable this, and keep this disabled by default? I am concerned this will also create load.

Reply:
you mean to disable this whole logging part (and sentry) in case the status is not valid for sending an email?

Reply:
I meant just this part: sentry when unexpected status is returned. 
But, to be safe, let's just disable this part entirely. 
Only log, not sentry, if other status than "submitted" is returned.